### PR TITLE
Create feature file of card and list

### PR DESCRIPTION
**Pull request by JuanCarlosMartinez4**

### CODE REVIEW

File: trello/api/features/acceptance/CardAcceptance.feature [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
   As a regular user, it wants to manage a card, and creates a card.
+
+  Scenario: Add a new card with name
+    Given Sets a "POST" request to "/boards/"
+      | key  | value                |
+      | name | boardForCardWithName |
+    And Sends request
+    And Should return status code 200
+    And Saves response as "BoardObject"
+    When Sets a "POST" request to "/lists/"
```
Comment (Amet015) [inline trello/api/features/acceptance/CardAcceptance.feature]: are you testing a List? why WHEN is here?

File: trello/api/features/acceptance/CardAcceptance.feature [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+    And Should return status code 200
+    And Saves response as "ListObject"
+    And Sets a "POST" request to "/cards/"
+      | key    | value           |
+      | idList | (ListObject.id) |
+      | name   | newCardName     |
+    And Sends request
+    Then Should return status code 200
+    And Saves response as "CardObject"
+    And Should return status code 200
```
Comment (Amet015) [inline trello/api/features/acceptance/CardAcceptance.feature]: this line is repeat

File: trello/jsonfiles/card_label_schema.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    }
+  },
+  "required": [
+    "id",
+    "idBoard",
+    "name",
+    "color",
+    "limits"
+  ]
+}
```
Comment (Amet015) [inline trello/jsonfiles/card_label_schema.json]: add new line at the end

File: trello/api/features/acceptance/CardAcceptance.feature [hunk 4]
```diff
@@ -38,4 +71,4 @@ Feature: Card
     And Should return status code 200
     And Sets a "DELETE" request to "/boards/BoardObject.id"
     And Sends request
-    And Should return status code 200
+    And Should return status code 200
```
Comment (Enrique-C) [inline trello/api/features/acceptance/CardAcceptance.feature]: add a code line at the last

File: trello/api/features/functional/CardFuntional.feature [hunk 5]
```diff
@@ -84,40 +84,83 @@ Feature: Card
...
+
+
+  Scenario: Add a new comment to a card
+    Given Sets a "POST" request to "/boards/"
+      | key  | value               |
+      | name | boardForCardComment |
+    And Sends request
+    And Should return status code 200
+    And Saves response as "BoardObject"
+    When Sets a "POST" request to "/lists/"
```
Comment (Enrique-C) [inline trello/api/features/functional/CardFuntional.feature]: Review this "When"

File: trello/api/features/functional/CardFuntional.feature [hunk 6]
```diff
@@ -84,40 +84,83 @@ Feature: Card
...
+      | idBoard | (BoardObject.id) |
+      | name    | cardList         |
+    And Sends request
+    And Should return status code 200
+    And Saves response as "ListObject"
+    And Sets a "POST" request to "/cards/"
+      | key    | value           |
+      | idList | (ListObject.id) |
+    And Sends request
+    Then Should return status code 200
```
Comment (Enrique-C) [inline trello/api/features/functional/CardFuntional.feature]: It wouldn't be "Then"

File: trello/api/features/functional/CardFuntional.feature [hunk 7]
```diff
@@ -84,40 +84,83 @@ Feature: Card
...
+    And Saves response as "CardObject"
+    And Should return status code 200
+    And Sets a "POST" request to "/cards/CardObject.id/actions/comments"
```
Comment (Enrique-C) [inline trello/api/features/functional/CardFuntional.feature]: It would be "When"

File: trello/api/features/functional/CardFuntional.feature [hunk 8]
```diff
@@ -84,40 +84,83 @@ Feature: Card
...
+      | key  | value             |
+      | text | This is a comment |
+    And Sends request
+    And Should return status code 200
```
Comment (Enrique-C) [inline trello/api/features/functional/CardFuntional.feature]: It would be "Then"

**Review by Enrique-C — APPROVED** [review]

**Review by Amet015 — APPROVED** [review]

**Pull request merged**